metals/src/main/scala/scala/meta/internal/metals/Configs.scala

```diff
@@ -72,6 +72,10 @@ object Configs {
           "metals.hover.documentation",
           default = true
         ),
+        isMagicIndentClient = MetalsServerConfig.binaryOption(
+          "metals.magic-indent-client",
```

Comment:
I'm not a fan of the "magic indent" name. How about `metals.snippet-vscode-indent=true`? The behavior of the option would be determined by how VS Code treats indentation in completion `new Text` snippets.

Reply:
Sure, I can change that. One argument to use magic-indent though is that it follows other patters such as the one used here https://github.com/spring-projects/sts4/blob/master/headless-services/commons/commons-language-server/src/main/java/org/springframework/ide/vscode/commons/languageserver/completion/VscodeCompletionEngineAdapter.java#L287-L289 and also language that has already been used when talking about the issue on the LSP GitHub page. I'm just worried that having vscode in the name is very vscode-centric when it's not just vscode that does it. For example, atom also uses the same concept.